crates/compiler/fmt/src/pattern.rs

```diff
@@ -42,7 +42,8 @@ impl<'a> Formattable for Pattern<'a> {
         // Theory: a pattern should only be multiline when it contains a comment
         match self {
             Pattern::SpaceBefore(_, spaces) | Pattern::SpaceAfter(_, spaces) => {
-                debug_assert!(!spaces.is_empty());
+                //TODO: This isn't always true, should it be?
+                // debug_assert!(!spaces.is_empty());
```

Comment:
Do you have a reproducer of when it's not? It should never be true. If there's a counterexample can you create an issue?

Reply:
It was actually crashing every few minutes because of this. I'll see if I can get it to log out some details

Reply:
I got a reproduction and made an issue https://github.com/roc-lang/roc/issues/6521